src/logger/logger.service.ts

```
@@ -1,38 +1,124 @@
+    const [context] = optionalParams as [string?];
+    this.logger.log({ level: 'info', message, context });
   }
 
-  error(message: string, trace: string, context?: string) {
+  error(message: any, ...optionalParams: any[]) {
+    const [trace, context] = optionalParams as [string?, string?];
     this.logger.error({ message, trace, context });
+    this.sendToSentry('error', String(message), context, trace);
   }
 
   warn(message: string, context?: string) {
```

ESLint

[error] Replace `··` with `↹`

(prettier/prettier)